tgui/packages/tgui/interfaces/PreferencesMenu/SpeciesPage.tsx

```diff
@@ -259,33 +259,33 @@ const SpeciesPageInner = (props: {
       <Stack.Item grow>
         <Stack fill>
           <Stack.Item>
-            <Stack vertical fill>
+            <Box height="calc(100vh - 170px)" overflowY="auto" pr={3}>
```

Comment:
What's the reason for this magic dynamic height?

Reply:
could not for the life of me remember how to get it to grow to fit the end of the *screen*

Reply:
the 170px comes from the top bar, which contains the list of characters, tabs, etc